pkg/api/validation/v1beta1-swagger.json

```diff
@@ -1312,6 +1312,31 @@
      }
     }
    },
+   "v1beta1.ISCSI": {
```

Comment:
I offered a PR just for base swagger stuff.  Include it in your PR and
whoever loses can rebase.

On Thu, Apr 2, 2015 at 6:23 AM, Huamin Chen <notifications@github.com>
wrote:

> In pkg/api/validation/v1beta1-swagger.json
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5506#discussion_r27655601>
> :
>
> > @@ -1312,6 +1312,31 @@
> >       }
> >      }
> >     },
> > +   "v1beta1.ISCSI": {
>
> I ran ./hack/update-swagger-spec.sh but it generated more stuff than just
> iSCSI (see below). Should I check in the those generated files?
>
> http://pastebin.com/LTe0kRgw
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5506/files#r27655601>
> .
>